src/community/china/index.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Using Flutter in China
+description: Where to find a version of the Flutter site that is localized to Simplified Chinese.
+toc: true
+os-list: [Windows, macOS, Linux, ChromeOS]
+---
+
+{% assign path = 'flutter_infra_release/releases/stable/windows/flutter_windows_3.3.0-stable.zip' -%}
+
+To speed the download and installation of Flutter in China,
+consider using a mirror site.
+
+{{site.alert.important}}
+  Use mirror sites only if you _trust_ the provider.
+  The Flutter team cannot verify their reliability or security.
+{{site.alert.end}}
+
+## Use a Flutter mirror site
+
+The [China Flutter User Group][] (CFUG) maintains a Simplified Chinese
+Flutter website [https://flutter.cn](https://flutter.cn) and a mirror.
+Other mirrors can be found at the [end of this guide](#known-trusted-community-run-mirror-sites).
+
+### Configure your machine to use a mirror site
+
+To install or use Flutter in China, use a trustworthy Flutter mirror site.
+This requires setting two environment variables on your machine.
+
+_All examples that follow presume that you are using the CFUG mirror._
+
+To set your machine to use a mirror site:
+
+{% comment %} Nav tabs {% endcomment -%}
+<ul class="nav nav-tabs" id="china-os-tabs" role="tablist">
+{% for os in page.os-list %}
+{% assign id = os | downcase -%}
+  <li class="nav-item">
+    <a class="nav-link {%- if id == 'windows' %} active {% endif %}" id="{{id}}-tab" href="#{{id}}" role="tab" aria-controls="{{id}}" aria-selected="true">{{os}}</a>
+  </li>
+{% endfor -%}
+</ul>
+
+{% comment %} Tab panes {% endcomment -%}
+<div class="tab-content">
+{% include_relative _os-settings.md os="Windows" %}
+{% include_relative _os-settings.md os="macOS" %}
+{% include_relative _os-settings.md os="Linux" %}
+{% include_relative _os-settings.md os="ChromeOS" %}
+</div>
+
+### Download Flutter archives based on a mirrored URL
+
+To download Flutter archives from a mirrored URL,
+replace `storage.googleapis.com` with the URL of your trusted mirror.
+This should improve download speed and the replaced URL
+can be used to download other software.
```

Comment:
```suggestion
can be used to download in other software.
```

Reply:
"download in" wouldn't work here because there's no target given. Leaving as is.

Reply:
Ah what I mean is use other softwares with the URL to download artifacts